### PR TITLE
[TF] More fixes for GCP DNS

### PR DESCRIPTION
**Pull request by sionescu**
### Description

* add variable to control the record TTL and lower the default from 1h to 5min, as the rest of the records on GCP
* add variable *create_dns_records* to the testnet module as well, to be useable from the devnet module

### Test Plan

Tested on the new GCP Devnet.

### CODE REVIEW

**Review by rustielin — APPROVED** [review]
do you also still need this https://github.com/aptos-labs/aptos-core/pull/8160 in conjunction with this PR?

**Comment by rustielin** [comment]
@sionescu I think it makes sense in a world where we don't care about DNS but still want a testnet or single node. It's mostly for test cases, and also brings it up to parity to the AWS setup

**Comment by sionescu** [comment]
@rustielin Yes, in that case let's merge #8160.

**Review by sherry-x — APPROVED** [review]

File: terraform/helm/testnet-addons/templates/ingress.yaml [hunk 1]
```diff
@@ -5,15 +5,17 @@ metadata:
   labels:
     {{- include "testnet-addons.labels" . | nindent 4 }}
   annotations:
+    {{- if .Values.service.domain }}
+    external-dns.alpha.kubernetes.io/hostname: {{ .Values.service.domain }}
+    {{- end }}
+    # EKS annotations
+    {{- if eq .Values.cloud "EKS" }}
```
Comment (rustielin) [inline terraform/helm/testnet-addons/templates/ingress.yaml:12]: nit: `aws` and `gcp` rather than `EKS` and `GKE`?

**Review by rustielin — CHANGES_REQUESTED** [review]

**Review by rustielin — APPROVED** [review]

**Pull request merged**